modules/quality_control/jsx/columnFormatterImg.js

```diff
@@ -0,0 +1,43 @@
+/**
+ * Modify behaviour of specified column cells in the Data Table component
+ * @param {string} column - column name
+ * @param {string} cell - cell content
+ * @param {array} rowData - array of cell contents for a specific row
+ * @param {array} rowHeaders - array of table headers (column names)
+ * @return {*} a formated table cell for a given column
+ */
+function formatColumn(column, cell, rowData, rowHeaders) {
```

Comment:
I don't think this should be a global function, it should be a method on the class which is using it. The `formatColumn` function convention predates us converting most things to react, when there was no class to own the function, and doesn't make sense for new modules which are written entirely in React